foundation/foundation-mda/src/main/resources/templates/data-delivery-pyspark/asynchronous.processor.base.py.vm

```diff
@@ -12,10 +12,11 @@ from aiops_core_filestore.file_store_factory import FileStoreFactory
 #if (!$step.provenance || $step.provenance.enabled)
 from aiops_core_metadata.hive_metadata_api_service import HiveMetadataAPIService
 #end
-#if ($step.isPersistTypePostgres() || $step.isPersistTypeRdbms())
+#if ($step.isPersistTypePostgres())
 from aiops_core_config import SparkPostgresConfig
```

Comment:
Q/S: Why are we still persisting the notion of Postgres as a distinct option separate from general RDBMS offerings?  My impression is that the SparkPostgresConfig has negligible functionality associated with it, and the RDBMS config should be able to fully replace it.

Reply:
At the time of implementation I was being cautious not to pull the postgres configs that could still be being used in downstream projects under the impression it'd be best to save that for a baton migration follow-up. 

I think you're right though - since most of the changes ended up being to base files and it's unlikely the jdbc configs class will be referenced outside of them, there's probably no need for a migration and it'd be best to replace the postgres configs entirely.

Reply:
I would note that the `Postgres` persist type has been deprecated for a significant amount of time, and has not been present in our official documentation for the same amount of time.  In this case, I think it's safe to remove, especially since if it's entirely in base/generated code.  Strictly speaking, we don't have to do it _right now_, but the amount of changes in this PR that go out of their way to maintain explicit Postgres support as a distinct entity does concern me.

Reply:
@peter-mcclonski I was able to pull out all the postgres persist type code from the PySpark implementation. I noticed the Spark/Java side also makes an effort to preserve the postgres option, but I take it that's out of scope for this specific issue.